source/jormungandr/jormungandr/parking_space_availability/abstract_provider_manager.py

```diff
@@ -89,7 +89,7 @@ def _find_provider(self, poi):
     def _init_class(self, cls, arguments):
         try:
             if '.' not in cls:
-                self.log.warn('impossible to build, wrongly formated class: {}'.format(cls))
```

Comment:
You missed one just below, line 99 :wink: 
(don't know if it's worth a new pass with the regexp for other .err() or so)

Reply:
Thanks :+1: , don't know where it was in my RipGrep response, :thinking:
probably a cache problem. 
It seems it was the only one

Reply:
recheck for err too